connector.go

```diff
@@ -264,8 +263,8 @@ func (c *Connector) makeRequest(t RequestType, obj IBObject, ref string) (res []
 
 func (c *Connector) CreateObject(obj IBObject) (ref string, err error) {
 	ref = ""
-
-	resp, err := c.makeRequest(CREATE, obj, "")
+	var forcedProxy bool
```

Comment:
It would be much clearer doing `forcedProxy := false` instead.

Reply:
done